docs/advanced/command-line/commands.md

```diff
@@ -0,0 +1,42 @@
+# Commands
+
+## Launch Datapreparer
+* Linux or Mac: ./datapreparer.sh
+* Windows: datapreparer.bat
+
+## Command parameters:
+* -b or –no-browser. The default behaviour in Data Preparer is that the home page of the
+web application, accessible at http://localhost:8182, is launched in the default browser.
+The -b parameter, however, prevents the browser from starting up, so as to complete a
+wrangling process purely in the command line. This could be used in conjunction with
+the -x option that exports the result to a csv file, or when the end product is written to a
+database. In the presence of the -b parameter, Data Preparer will exit having run a wrangling process on its inputs.
+
+* -d or –data-context filename: filename is a path to a file that describes where the resources
+in the data context are.
+
+* -e or –export-scenario filename: This will export filename, that is a Data Preparer scenario
+file, a compressed file containing the configuration options, the sources file, the data context file, the feedback provided by the user, and any csv files that have been uploaded from the web interface. This scenario file can subsequently be imported using the -i parameter, so that you can resume working with the scenario, or share it.
+
+* -f or –feedback filename: filename is a path to a file that contains feedback instances.
+
+* -h or –help: Prints a help message with the available command line options and exits.
+
+* -i or –import-scenario filename: This initialises a wrangling scenario by loading the filename, which is a compressed file containing one or more of the following: a properties file with the wrangling configuration, a sources.csv file with information about how to access the sources, a datacontext.csv file with the information about how to access resources in the data context, a feedback.csv file with feedback instances, and finally any csv files that may have been uploaded from the web interface. This is complementary to the -e option that exports the scenario configuration.
+
+* -l or –license-key provides the license key to Data Preparer; a license key is emailed to the email address provided when downloading the software. The argument in this parameter provides either the license key itself or the path to a file containing the license key. In order not to have to specify the license key every time Data Preparer runs, a file containing ’license’ in its name, can be placed in the same folder with the executable, containing the license key.
+
+* -o or–override property=value. This loads the scenario and overrides one or more properties in the configuration. For instance, -o product.name=p1,product.size=200 will set the product name to p1 and the number of tuples to include in the end product to 200. 
+> Note that no spaces are allowed in the property-value pairs.
```

Comment:
Format would be better if it was Note: no spaces allowed... etc.

Reply:
Done!